manager-bundle/tests/HttpKernel/ContaoKernelTest.php

```diff
@@ -274,6 +284,102 @@ public function containerConfigurationProvider(): iterable
         ];
     }
 
+    public function testAutowiresSrcFiles(): void
+    {
+        // Preload for autoloading
+        include_once __DIR__.'/../Fixtures/HttpKernel/AutowireSrc/src/Entity/FooEntity.php';
+        include_once __DIR__.'/../Fixtures/HttpKernel/AutowireSrc/src/EventListener/InvalidListener.php';
+        include_once __DIR__.'/../Fixtures/HttpKernel/AutowireSrc/src/EventListener/ValidListener.php';
+        include_once __DIR__.'/../Fixtures/HttpKernel/AutowireSrc/src/FrontendModule/LegacyModule.php';
+        include_once __DIR__.'/../Fixtures/HttpKernel/AutowireSrc/src/Messenger/UnionTypeMessage.php';
+        include_once __DIR__.'/../Fixtures/HttpKernel/AutowireSrc/src/Model/FooModel.php';
```

Comment:
It would be a lot easier if you just used the correct namespaces like we do in the core bundle:

https://github.com/contao/contao/blob/048082b849ab077344ecb2eb047d0394695255fb/core-bundle/tests/Fixtures/Entity/Author.php#L13-L21

Then you would not need an ignore-rule for the depcheck, either.

Reply:
I meant adding `"App\\": "manager-bundle/tests/Fixtures/HttpKernel/AutowireSrc/src/"` to the composer.json file, but that has drawbacks, too, so let's just keep it this way.